docs/rules/no-invalid-template-root.md

```diff
@@ -6,16 +6,21 @@ This rule checks whether every template root is valid.
 
 This rule reports the template root if the following cases:
```

Comment:
I know it's out of the scope of this task, but if you're already on it maybe you could change `if` to `in`. We can also do a separate PR with copy updates though, so whatever..